CHANGELOG.md

```diff
@@ -73,6 +74,20 @@
 - [0.2.0](#020)
 - [0.1.0](#010)
 
+## [v0.60.0]
```

Comment:
Please add a link for this at the very bottom of this document https://github.com/Kong/go-kong/blob/6b65f06abd6af3fd82c5a9ff95f24ddccbc7ed50/CHANGELOG.md?plain=1#L968 so that viewers can click it and see the diff between this one and previously released version.